SendGrid/src/SendGrid/project.json

```diff
@@ -0,0 +1,48 @@
+{
+  "version": "1.0.1-beta-*",
+  "authors": [ "SendGrid" ],
+  "description": "C# client library and examples for using SendGrid API's to send mail and access Web API v3 endpoints.",
+  "packOptions": {
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/sendgrid/sendgrid-csharp"
+    },
+    "tags": [
+      "sendgrid",
+      "email",
+      "microsoft",
+      "azure"
+    ]
```

Comment:
here your package description file, the *.nupkg file just placed in bin/Release | Debug folder